lab2.js

```diff
@@ -42,31 +36,49 @@ function assert(expression, failureMessage) {
     second hour     |    2/hour        |        2         |
     third hour      |    3/hour        |        3         |
     fourth hour     |    4/hour        |        4         |
-
  TODO: First, make a constructor function, called Blob, that makes blobs.
-
  TODO: Next, create an instance of Blob named blob.
-
  TODO: Then, use a loop to calculate how long it took the blob to finish
  with Dowington. */
 
+function Blob(blobs) {
+  return blobs;
+}
 
-var hoursSpentInDowington; // TODO: assign me the value of the
-                           // above calculation (how long it took
-                           // the blob to eat Dowington)
+var blob = new Blob();
+
+var peopleLeft = 1000;
+var j = 1;
+while (peopleLeft > 0) {
+  peopleLeft = (peopleLeft - j);
+  j++;
+}
+var hoursSpentInDowington = j;
+console.log(hoursSpentInDowington);
 
 // Now, write a method that takes a population for an arbitrary
 // town, and the starting consumption rate, and returns the number
 // of hours the blob needs to ooze its way through that town.
 
-function hoursToOoze(population, peoplePerHour) {
+Blob.prototype.hoursToOoze = function(population, peoplePerHour) {
+  this.population = population;
+  this.peoplePerHour =  peoplePerHour;
+  for (var i = 0; this.population > 0; i++) {
+    this.population = this.population - (this.peoplePerHour * i);
+  }
+  var hoursSpentInDowington = i;
+  console.log(hoursSpentInDowington);
```

Comment:
this should be return; right now your asserts aren't passing because there's no value being returned out of the function in the asserts.